helpers/award_helper.py

```diff
@@ -25,7 +25,7 @@
 AWARD_MATCHING_STRINGS = [
     (AwardType.CHAIRMANS, (["chairman"], ["hon", "finalist"])),
     (AwardType.CHAIRMANS_HONORABLE_MENTION, (["chairman", "hon", "mention"], [])),
-    (AwardType.CHAIRMANS_HONORABLE_MENTION, (["chairman", "finalist"], [])),
+    (AwardType.CHAIRMANS_FINALIST, (["chairman", "finalist"], ["hon", "mention"])),
```

Comment:
It's the blacklist for searching, so I figured that if "hon" and "mention" were going to be for `CHAIRMANS_HONORABLE_MENTION`, you wouldn't want them to match for `CHAIRMANS_FINALIST`.
I'm totally open to changing this, I was just trying to make sure you'd get the right results when searching.